frontend/src/models/VideoStream.ts

```diff
@@ -1,8 +1,17 @@
-export interface VideoStream {
-    id: string
-    name: string
-    robotId?: string
+export enum MediaType {
+    Video,
+    Audio,
+}
+
+export enum MediaConnectionType {
+    LiveKit = 'LiveKit',
+}
+
+export interface MediaStreamConfig {
     url: string
-    type: string
-    shouldRotate270Clockwise: boolean
+    token: string
+    robotId: string
+    mediaConnectionType: MediaConnectionType
 }
+
+export type MediaStreamConfigAndTracks = MediaStreamConfig & { streams: MediaStreamTrack[] }
```

Comment:
Consider splitting this into two objects to avoid needing to naming it as Thing1AndThing2

Reply:
I am not quite sure I understand what you mean. Is it the name you disagree with or the expression on the right?

Reply:
I think the name is good as it describes well the expression on the right. Though when the name includes *And* i suspect that it might be better suited as two distinct entities that is better combined when they are used. Though I am not familiar enough with webrtc to know if it makes good sense to combine it or if it is better to combine where needed

Reply:
Have simplified this type.